hca/dss/__init__.py

```diff
@@ -38,19 +43,30 @@ def download(self, bundle_uuid, replica, version="", dest_name="",
         """
         Download a bundle and save it to the local filesystem as a directory.
 
-        `metadata_files` (`--metadata-files` on the CLI) are one or more shell patterns against which all metadata
-        files in the bundle will be matched case-sensitively. A file is considered a metadata file if the `indexed`
-        property in the manifest is set. If and only if a metadata file matches any of the patterns in
-        `metadata_files` will it be downloaded.
-
-        `data_files` (`--data-files` on the CLI) are one or more shell patterns against which all data files in the
-        bundle will be matched case-sensitively. A file is considered a data file if the `indexed` property in the
-        manifest is not set. If and only if a data file matches any of the patterns in `data_files` will it be
-        downloaded.
+        :param str bundle_uuid: The uuid of the bundle to download
+        :param str replica: the replica to download from. The supported replicas are: `aws` for Amazon Web Services, and
+            `gcp` for Google Cloud Platform. [aws, gcp]
+        :param str version: The version to download, else if not specified, download the latest. The version is a
+            timestamp of bundle creation in RFC3339
+        :param str dest_name: The destination file path for the download
+        :param list metadata_files: one or more shell patterns against which all metadata files in the bundle will be
+            matched case-sensitively. A file is considered a metadata file if the `indexed` property in the manifest is
+            set. If and only if a metadata file matches any of the patterns in `metadata_files` will it be downloaded.
+        :param list data_files: one or more shell patterns against which all data files in the bundle will be matched
+            case-sensitively. A file is considered a data file if the `indexed` property in the manifest is not set. If
+            and only if a data file matches any of the patterns in `data_files` will it be downloaded.
+        :param int initial_retries_left: The initial quota of download failures to accept before exiting due to
```

Comment:
This parameter should be renamed to something like `num_retries`. "Initial" and "left" are implementation details.

"The number of times to attempt the download before exiting with an error"